build/configs/qemu/README.md

```diff
@@ -4,7 +4,9 @@ QEMU is a generic and open source machine emulator and virtualizer.
 
 ## Information
 
-QEMU uses lm3s6963-ek machine.
+QEMU and TizenRT support the lm3s6965-ek board.
```

Comment:
Do you want to show it in same line?
If you want to show it in other line, you should add two spaces at end of line.

Reply:
I fixed it